package.json

```diff
@@ -18,7 +18,8 @@
     "newrelic": "^6.6.0",
     "probot": "^9.11.3",
     "probot-scheduler": "^2.0.0-beta.1",
-    "probot-stale": "github:oppia/oppiabot-stale"
+    "probot-stale": "github:oppia/oppiabot-stale",
+    "probot-config": "^1.1.0"
```

Comment:
@ankita240796 I'm not sure where probot-config is used.

Reply:
It is a dependency for both both probot, probot-stale. By default, they are using an old version 0.1.0 which results in issues like #60 because of deprecated functions. (Note: for probot stale, update isn't done in this PR).

Reply:
Ohh
Okay. Thanks for the explanation.